Readme.md

```diff
@@ -1,75 +1,26 @@
-# Today's Objective: A personal NPM library
+# shocked-pikachu
+ inserts a pikachu asci art into your react component
 
```

Comment:
This README is succinct and effective. My only suggestion would be to maybe include a screenshot example of what Pikachu will look like?